TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -0,0 +1,259 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2013-2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mbed.h"
+#include "greentea-client/test_env.h"
+#include "utest/utest.h"
+#include "unity/unity.h"
+
+
+using namespace utest::v1;
+
+#define TestTicker LowPowerTicker
```

Comment:
I don't like that, it's confusing. Is there a particular reason you are doing that?